### PR TITLE
PREAPPS-7532: Distribution list is missing on choose contact popup

### PR DESCRIPTION
**Pull request by sandesh-telus**
- Added new query to fetched distribution lists details

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -934,6 +934,18 @@ export class ZimbraBatchClient {
...
+		this.jsonRequest({
+			name: 'GetAccountDistributionLists',
+			body: {
+				attrs,
+				ownerOf
+			},
+			namespace: Namespace.Account
+		}).then(res => {
+			return { dls: res?.dl ? res.dl : [] };
+		});
```
Comment (lavegupta) [inline src/batch-client/index.ts:947]: `}).then(res => ({ dls: res?.dl || [] }));`

Comment (sandesh-telus) [reply src/batch-client/index.ts]: Done

Comment (miteshsavani810) [reply src/batch-client/index.ts]: Make it inline statement. as per suggestion it is still missing.

Comment (sandesh-telus) [reply src/batch-client/index.ts]: @miteshsavani810 sorry I missed that point. I have make it inline statement. Please verify.

**Review by miteshsavani810 — APPROVED** [review]

**Review by lavegupta — APPROVED** [review]

**Pull request merged**